JumpscaleCore/clients/threebot/ThreebotClientFactory.py

```diff
@@ -22,30 +29,32 @@ def explorer_addr(self):
         else:
             return j.core.myenv.config["EXPLORER_ADDR"] + ""
 
-    def explorer_addr_set(self, value):
-        """
+    def explorer_public_key(self):
+        if "EXPLORER_PUBKEY" not in j.core.myenv.config:
+            return None
+        else:
+            return j.core.myenv.config["EXPLORER_PUBKEY"] + ""
 
-        :param value:
-        :return:
-        """
-        j.core.myenv.config["EXPLORER_ADDR"] = value
+    def explorer_addr_set(self, addr, pubkey):
```

Comment:
probably should check against the defined `_threefold_explorer_public_keys` if pubkey is None

Reply:
I do that in the explorer property just underneath. do you prefer it here ?

Reply:
it's just cumbersome to paste a known address when you use `explorer_addr_set("explorer.testnet.grid.tf")` because it requires the pubkey everytime, I think it's fair to ignore pubkey if it's for a known explorer domain :)